TimeOfFlight/src/TOFEstimators.cc

```diff
@@ -15,7 +12,20 @@
 #include <UTIL/Operators.h>
 #include <UTIL/PIDHandler.h>
 
-#include "DDRec/Vector3D.h"
+#include "EVENT/LCCollection.h"
+#include "EVENT/ReconstructedParticle.h"
+using EVENT::LCCollection, EVENT::ReconstructedParticle;
```

Comment:
This does not work!
```
error: expected ';' before ',' token
 using EVENT::LCCollection, EVENT::ReconstructedParticle;
```

Reply:
That is interesting. It compiles successfully in the CI workflows. But maybe that is only true for the limited number of compilers that we use there. Where did you test this?

Reply:
one of the pipelines that creates the cvmfs nighly https://gitlab.cern.ch/CLICdp/SoftwareConfigurations/iLCSoft/-/pipelines/2360644

Reply:
This valid c++17 code that gcc 6 does not seem to support. Do we still need gcc6 to work here ?

Reply:
@andresailer has to decide if he still wants that pipeline. But in my opinion it's easier to fix this and keep the current system running.

Reply:
Are we moving to c++17 for iLCSoft?

Reply:
I thought we did a long time ago ? At least we are using it for our iLCSoft release > 15 months or so ...

Reply:
I must have missed that memo 😃 

Probably should move to base on the nightly key4hep installation for the CI, which includes the iLCSoft packages

Reply:
So we stop the old nightly CI in [this](https://gitlab.cern.ch/CLICdp/SoftwareConfigurations/iLCSoft/pipelines) project?

Reply:
yes